spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -45,6 +45,7 @@
     then_I_can_see_a_summary_section
     and_I_can_edit_the_summary_section
     and_I_can_see_a_sidebar_settings_section_with_link "Edit"
+    and_I_can_see_a_secondary_links_section_with_link "Edit"
```

Comment:
Same here, I think there should be a step after this that checks that when the "Edit" link is clicked, the user is taken to the "Secondary content" page, so we can be sure that we're linking to the right place.